python-shaptools.spec

```diff
@@ -67,5 +67,6 @@ API to expose SAP HANA functionalities
 %license LICENSE
 %endif
 %{python_sitelib}/*
+%{_bindir}/shapcli
```

Comment:
Based on the FHS, I think the right location for this is `/usr/bin` as it is not essential.

https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard

Reply:
%{_bindir} is installing the file in /usr/bin, isn't. At least this is how it works in my system using Leap 15

Reply:
@arbulu89 Hum... you are right, but the setup.py points to bin/shapcli, is that correct?

Reply:
setup.py points to the file that will be moved to /usr/bin (so in our repo is stored in bin/shapcli)